DEVELOPING.md

```diff
@@ -0,0 +1,86 @@
+# Python
+
+This project uses `A | B` syntax for type unions in type annotations, which requires Python 3.10+.
+
+# Native deps
+
+Some Python packages assume that corresponding native dependencies are already present.
+
+## MacOS
+
+```
+brew install freetype imagemagick
+```
+
+# virtualenv
+
+You should use a virtualenv, through your preferred version management tool,
+for example pyenv or uv.
+
+## uv (optional)
+
+uv is an all-in-one Python environment manager.
+Our current configuration permits, but does not require, the use of uv.
+To use uv, prefix Python-related commands with:
+
+```
+uv run
+```
+
+For example:
+
+```
+uv run python
+```
+
+# Install this package as editable
+
+## uv
+
+```
+uv pip install --editable .
+```
+
+## virtualenv (no uv)
+
+```
+source .venv/bin/activate
+pip install --editable .
```

Comment:
Should we change this to `'.[dev]'`? So people get development tools?

Reply:
Probably, yeah.

Reply:
```suggestion
pip install --editable '.[dev]'
```

Reply:
Oh, wait. Bonus interaction: Square brackets are shell special chars.